app/routers/profile.py

```diff
@@ -148,3 +149,64 @@ def get_image_crop_area(width, height):
         return (delta, 0, width - delta, height)
     delta = (height - width) // 2
     return (0, delta, width, width + delta)
+
+
+@router.post("/update_holidays")
+async def update_holidays(
+        file: UploadFile = File(...), session=Depends(get_db)):
```

Comment:
Is it possible to validate the file format?

Reply:
Do you mean to check if the file has ics extension or validate the file itself for valid ics file? for the first one we have the input accept=".ics" but i guess i could check it after upload as well. for the later, as its text file and im using regex, if i wont have the valid format im looking for ill get 0 results.